browser-test/utils.js

```diff
@@ -1,34 +1,31 @@
-import {spawn} from "child_process";
+import {Builder} from "selenium-webdriver";
+import {spawnSync} from "child_process";
 import testData from "./testData.json";
 
+const firefox = require("selenium-webdriver/firefox");
+
 const waitTime = 5000;
 const orgSlug = "default";
 
-export const executeCommand = async (command, argv) => {
-  const executor = await spawn(command, [argv]);
-  executor.stdout.on("data", (data) => {
-    process.stdout.write(`${data}\n`);
-  });
-
-  executor.stderr.on("data", (data) => {
-    process.stderr.write(`stderr: ${data}\n`);
-  });
-
-  executor.on("error", (error) => {
-    process.stderr.write(`error: ${error.message}\n`);
-  });
-
-  executor.on("close", (exitCode) => {
-    if (exitCode !== 0) process.exit(exitCode);
-  });
+export const executeCommand = (command, argv) => {
+  const result = spawnSync(command, [argv]);
+
+  if (result.stdout) {
+    process.stdout.write(`${result.stdout}\n`);
+  }
+  if (result.stderr) {
+    process.stderr.write(`${result.stderr}\n`);
+  }
+  if (result.status !== 0) {
+    process.exit(result.status);
+  }
```

Comment:
@codesankalp unfortunately I am no expert when it comes to nodeJS.
While testing I have noticed that the scripts keep running after the tests have finished and jest complains about it.
After debugging I found out that we could have used `spawnSync` which is a lot simpler and more suited for this kind of operation.

I also have seen there's `execSync` but would have probably meant to change a lot more the code so I didn't go for it.